middleware/redirects.ts

```diff
@@ -8,4 +9,8 @@ export default function ({ store, redirect, route }): void {
       return redirect(`/${prefix}/explore/items`)
     }
   }
+
+  if (route.path.includes('/rmrk2/')) {
+    return redirect(route.path.replace('/rmrk2/', '/ksm/'))
```

Comment:
I would prefer to add the query parameter.

For now, If you open
`https://deploy-preview-5918--koda-canary.netlify.app/rmrk2/explore/items?listed=true&sort=updatedAt_DESC`, it will redirect you to `https://deploy-preview-5918--koda-canary.netlify.app/ksm/explore/items?listed=true` and the query `sort=updatedAt_DESC` would be lost.

Reply:
> I would prefer to add the query parameter.

oh thanks, updated